daily_snapshot_terraform/modules/daily_snapshot/service/calibnet_cron_job

```diff
@@ -3,4 +3,4 @@ cd "$BASE_FOLDER" || exit
 # shellcheck source=/dev/null
 source .bashrc
 flock -n /tmp/calibnet.lock -c "ruby daily_snapshot.rb calibnet > calibnet_log.txt 2>&1"
-flock -n /tmp/calibnet.lock -c "./upload_filops_snapshot.sh calibnet > filops_calibnet_log.txt 2>&1"
+flock -n /tmp/calibnet_filops.lock -c "./upload_filops_snapshot.sh calibnet > filops_calibnet_log.txt 2>&1"
```

Comment:
This PR implements a new feature. It shouldn't fix unrelated bugs at the same time. You can put the lock-file fix in another PR.